resources/js/Pages/Login.vue

```diff
@@ -75,12 +75,19 @@
       loginHandler(): void {
         try {
           const payload: string = this.getValidatedPayload();
-          this.$inertia.post('/login', payload, {
+          const test = this.$inertia.post('/login', payload, {
             replace: false,
             preserveState: true,
             preserveScroll: false,
-            _token: this.csrfToken
+            _token: this.csrfToken,
+            onSuccess: (res: any) => {
+              if ('error' in res.props) {
```

Comment:
i'm going to start using this instead of Object.hasOwnProperty()

Reply:
I wasn't sure if I should use that or hasOwnProperty, but using the in operator just felt cooler lol.